src/interfaces.rs

```diff
@@ -114,6 +114,11 @@ impl Interfaces {
     }
 
     pub(crate) fn get(&self, interface_name: &str) -> Option<&Interface> {
+        #[cfg(debug_assertions)]
+        if interface_name.ends_with(".json") {
+            warn!(interface_name, "the interface name passed ends in .json, this is commonly a BUG when the extension is copied from the interface file and shoulb probably be removed");
```

Comment:
What about something like this ?
```suggestion
            warn!(interface_name, "the interface name passed ends in \".json\", the input must be an interface name copied from the interface definition \"name\" field");
```

Reply:
I think the warning is more explicative of what happened if we just ask to remove the file extension

Reply:
```suggestion
            warn!(interface_name, "the interface name passed ends in .json, this is commonly a BUG when the extension is copied from the interface file and should probably be removed");
```